life_expectancy/cleaning.py

```diff
@@ -9,14 +9,14 @@
 
```

Comment:
In the second section of the assignment you have the following text "Right now you should have more python modules. If not, ensure you have at least 2: one for data cleaning and one for loading/saving data.". Note that you have only one module (`cleaning.py`) that both load/saves and cleans the data. In my opinion, no need to change, just a remark.